src/Rand.php

```diff
@@ -76,8 +74,7 @@ public static function getAlternativeGenerator()
         if (!class_exists('RandomLib\\Factory')) {
             throw new Exception\RuntimeException(
                 'The RandomLib fallback pseudorandom number generator (PRNG) '
-                . ' must be installed in the absence of the OpenSSL and '
-                . 'Mcrypt extensions'
+                . ' must be installed in the absence of a secure source'
```

Comment:
@weierophinney this is not cryptic as purpose, but definitely you find out a good point here. We can rephrase with "The RandomLib fallback pseudorandom generator is not installed. Please install it to support secure random numbers". What do you think?